File: LABA2/Laba2.2/LinearEquation.cpp
#include "LinearEquation.h"

LinearEquation::LinearEquation(double a, double b)
	: Solution(new double[2]{a, b}, 2)
{
		
}

int *LinearEquation::CalculateXResult()
{
	//return std::vector<double>(1, -GetRatio(0) / GetRatio(1));
	return 0;
}

File: LABA5/OOP_Laba5/LinearEquation.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace OOP_Laba5
{
	class LinearEquation : Solution
	{
		public  LinearEquation(double a, double b) 
			: base(new List<double>(2) {a, b})
		{
		}

		public override List<double> CalculateXResult()
		{
			return new List<double>(1) { -GetRatio(0) / GetRatio(1) };
		}
	}
}
